src/app/pages/backend-integration/diagram/backend-integration-diagram.component.html

```diff
@@ -0,0 +1,81 @@
+<div class='flex-column justify-content-center'>
+  <div class='flex-row justify-content-center header mt-auto mb-1'> What is admin dashboard with backend</div>
+  <div class='flex-row justify-content-center subheader'> You pay for backend connection (UI) + backend</div>
+  <div class='flex-row align-items-start justify-content-around mt-4 mb-auto'>
+    <div class='dashboard-container h-100 flex-column ml-3 mr-3'>
+      <div class='headers-container flex-column'>
+        <div class='header'> NGX-admin dashboard</div>
+        <div class='subheader'> Open source part + backend connection</div>
+      </div>
+      <div class='dashboard-content border-dashed pad1 flex-column justify-content-center'>
+        <div class='item mb-3'>
+          <div class='header'> REST Data Access</div>
+        </div>
+        <div class='flex-column border-dashed mb-3 p-3'>
+          <div class='item mb-3'>
+            <div class='header'> Nebular Components</div>
+          </div>
+          <div class='flex-row w-100'>
+            <div class='item flex-grow-1 mr-2'>
+              <div class='header'> Auth</div>
+            </div>
+            <div class='item flex-grow-1'>
+              <div class='header'> Security</div>
+            </div>
+          </div>
+        </div>
+        <div class='item mb-3'>
+          <div class='header'> Angular</div>
+        </div>
+      </div>
+    </div>
+    <div class='json-container mt-auto mb-auto header2 align-items-center'>
+      <nb-icon icon='arrow-back-outline'></nb-icon>
+      <div>JSON</div>
+      <nb-icon icon='arrow-forward-outline'></nb-icon>
+    </div>
+    <div class='backend-container h-100 flex-column ml-3 mr-3'>
```

Comment:
`h-100 flex-column ml-3 mr-3` are these classes come from bootstrap?

Reply:
Sure, you can find them [here ](dist/css/bootstrap.css)